blazor/common/data-binding/dapper-using-webapp.md

```diff
@@ -0,0 +1,570 @@
+---
+layout: post
+title: Bind data using Dapper with CRUD operations in Web App | Syncfusion
+component: "DataGrid component and DataManager"
+description: How to consume data from SQL database using Dapper, bind it to a Syncfusion Component, and perform CRUD operations in Blazor Web App.
+component: Common
+documentation: ug
+platform: Blazor
+---
+
+# How to Bind Data Using Dapper and Perform CRUD Operations
```

Comment:
sentence casing that is  camel casing need to follow in the titles. Please correct this for entire document

Reply:
As per discussion this file was removed from existing document.